src/cascadia/TerminalSettingsEditor/Launch.xaml

```diff
@@ -206,6 +195,65 @@
                                         Value="{x:Bind ViewModel.InitialRows, Mode=TwoWay}" />
                     </Grid>
                 </local:SettingContainer>
+
+                <!--  Launch Parameters  -->
+                <local:SettingContainer x:Uid="Globals_LaunchParameters"
+                                        CurrentValue="{x:Bind ViewModel.LaunchParametersCurrentValue, Mode=OneWay}"
+                                        Style="{StaticResource ExpanderSettingContainerStyle}">
+                    <Grid RowSpacing="10">
+                        <Grid.RowDefinitions>
+                            <RowDefinition Height="Auto" />
+                            <RowDefinition Height="Auto" />
+                            <RowDefinition Height="Auto" />
+                        </Grid.RowDefinitions>
+                        <Grid.ColumnDefinitions>
+                            <ColumnDefinition Width="*" />
+                            <ColumnDefinition Width="Auto" />
+                        </Grid.ColumnDefinitions>
+                        <TextBlock x:Uid="Globals_LaunchMode"
+                                   Grid.Row="0"
+                                   Grid.Column="0"
+                                   VerticalAlignment="Center" />
+                        <ComboBox Grid.Row="0"

```

Comment:
You might also need to set the [AutomationProperties.LabeledBy](https://learn.microsoft.com/en-us/uwp/api/windows.ui.xaml.automation.automationproperties.labeledby?view=winrt-22621#windows-ui-xaml-automation-automationproperties-labeledby) property on these controls? I've never used it before, but I assume this creates the association between the "header" and the "control".

Reply:
LabeledBy doesn't seem to do that haha, will just add name and help text manually